app/controllers/tag_controller.rb

```diff
@@ -40,12 +44,16 @@ def show
                   .page(params[:page])
                   .order('node_revisions.timestamp DESC')
     end
+
+      # breaks the parameter after its rub
```

Comment:
Can you align this comment and also, I'm not sure what you mean by "rub"? Thanks for the good documentation though!